### PR TITLE
bottle: fix inreplace string

### PR DESCRIPTION
**Pull request by dawidd6**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Fixes issue that popped up here: https://github.com/Homebrew/linuxbrew-core/pull/20803/checks?check_run_id=915097062

cc @vidusheeamoli

### CODE REVIEW

File: Library/Homebrew/dev-cmd/bottle.rb [hunk 1]
```diff
@@ -484,7 +484,7 @@ def merge
             update_or_add = "update"
             if args.keep_old?
               mismatches = []
-              bottle_block_contents = s[/  bottle do(.+?)end\n/m, 1]
+              bottle_block_contents = s.inreplace_string[/  bottle do(.+?)end\n/m, 1]
```
Comment (maxim-belkin) [inline Library/Homebrew/dev-cmd/bottle.rb:487]: Would it make sense to add the same number of spaces (2) before `end`?... just in case...

Comment (dawidd6) [reply Library/Homebrew/dev-cmd/bottle.rb:487]: Umm, it wasn't an issue before I think

**Comment by dawidd6** [comment]
Merging, as it's blocking other stuff.

**Comment by vidusheeamoli** [comment]
Thank you @dawidd6 🙏

**Pull request merged**